### ISSUE TITLE
Contact Import with Dropdown field(s) failing

### ISSUE THREAD

**Issue by adrianjean** [putyourlightson/craft-campaign#390]
**Description**:
- When importing a CSV file of contacts where the contacts have Dropdown fields set by values in the CSV, the import fails with the log flagging all Dropdown fields as the source of the failure

**Steps to reproduce:**
- Add one or more Dropdown fields to the contact Field layout with two or more possible options.
- Export a CSV from the Campaign->Contacts page to create a "template" to work from
- Add new contacts to the template, providing value(s) for dropdown fields for each new contact. Values are the not the "Title" but the actual "value" of the dropdown selection.
- Import the CSV and note that it fails, logs report the failure is on importing the Dropdowns

Craft: 4.4.13
Campaign: 2.7.1

**Configuration:**
- Dropdown: Gender Identity (`contactGender`)
  Value Options (Not titles): `female`, `male`, `other`

- Dropdown: Supporter (`contactSupport`)
  Value Options (Not titles): `yes`, `no`, `maybe`, `unknown` 

**Notes**
- I have MultiSelect fields  and Category fields that do seem to import.

**Sample CSV:**
```
email,contactFirstName,contactLastName,contactPhone,contactAddressStreet,contactAddressPostalCode,contactGender,contactGroup,contactSupporter,contactVolunteer,contactDonationAmount
test2@email.com,Sarah,Smith,555-222-1234,"235 Main St.","Z1Z 1Z1",female,Newsletter,yes,volunteer,400
```

**Import Log Error**
```
[2023-05-26 08:02:06] Line 1: Gender Identity is invalid.. Supporter is invalid.
```

**Comment by bencroker** [comment]
Thanks for reporting this. It turned out to be a bug in the importing of custom fields, meaning that the imported value for the dropdown field was blank which was likely not an allowed value. I’ve fixed it in https://github.com/putyourlightson/craft-campaign/commit/8574ef971ee834ae0edfc055b338645ee1202483 and released in version 2.7.2.

**Comment by adrianjean** [comment]
Amazing thank you! — Really outstanding plugin the more I dig into it!
